mirrord/operator/src/setup.rs

```diff
@@ -44,6 +44,7 @@ static OPERATOR_LICENSE_SECRET_FILE_NAME: &str = "license.pem";
 static OPERATOR_LICENSE_SECRET_VOLUME_NAME: &str = "license-volume";
 static OPERATOR_SERVICE_ACCOUNT_NAME: &str = "mirrord-operator";
 static OPERATOR_SERVICE_NAME: &str = "mirrord-operator";
+static OPERATOR_SECRET_ROLE_NAME: &str = "mirrord-operator-secret-access";
```

Comment:
```suggestion
static OPERATOR_ROLE_NAME: &str = "mirrord-operator";
```
plus rename exisiting `OPERATOR_ROLE_NAME` to `OPERATOR_CLUSTER_ROLE_NAME`